content/chronograf/v1.8/tools/chronograf-cli/_index.md

```diff
@@ -124,3 +124,15 @@ chronograf [flags]
 | `--generic-auth-url`      | Authorization endpoint URL for the OAuth 2.0 provider                          | `$GENERIC_AUTH_URL`      |
 | `--generic-token-url`     | Token endpoint URL for the OAuth 2.0 provider                                  | `$GENERIC_TOKEN_URL`     |
 | `--generic-api-url`       | URL that returns OpenID UserInfo-compatible information                        | `$GENERIC_API_URL`       |
+
+### etcd flags
+
+| Flag                      | Description                                                                                                | Env. Variable           |
+|:--------------------------|:-----------------------------------------------------------------------------------------------------------|:------------------------|
+| `-e`, `--etcd-endpoints`  | etcd endpoint URL (include multiple flags for multiple endpoints)                                          | `$ETCD_ENDPOINTS`       |
```

Comment:
Include direction for formatting multiple? 

Would this be an established standard across the CLI?

Reply:
It depends on the flag and which CLI you're using, but this has emerged as a pattern.